scripts/gateway-pre-start.sh

```
@@ -361,6 +392,132 @@ clawbox_stamp_read() {
+# `$include`: stood down, the way the core's own startup repair stands down —
+# the file on disk is not the whole config, so a preview built from it alone
+# would name keys an included file may already answer for.
+clawbox_core_residual_issues() {
+  local node_bin core_dist core_chunk preview verdict remembered
+
+  remembered="$(head -n1 "$CLAWBOX_POST_MIGRATION_ISSUES" 2>/dev/null || true)"
+  case "$remembered" in
+    "$(clawbox_config_fingerprint)$(printf '\t')"*) printf '%s' "${remembered#*$'\t'}"; return 0 ;;
+  esac
+
+  if grep -qF '"$include"' "$OPENCLAW_CONFIG" 2>/dev/null; then
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)